editors/document-model/editor-schema.tsx

```diff
@@ -68,13 +73,26 @@ export default function EditorSchema({
     ...props
 }: IProps) {
     const editorRef = useRef<editor.IStandaloneCodeEditor | null>(null);
+    const constrainedEditorRef = useRef<ConstrainedInstance | null>(null);
+    const modelEditorRef = useRef<editor.ITextModel | null>(null);
     const [code, setCode] = useState(value || '');
 
     useEffect(() => {
         const normalizedSchema = normalizeSchema(value, scope, name);
 
         if (normalizedSchema !== code) {
+            constrainedEditorRef.current?.removeRestrictionsIn(
```

Comment:
Temporarily turning off these restrictions so we can go ahead with changing the document name and renaming the state type.